addons/l10n_be/data/template/account.tax-be.csv

```diff
@@ -225,6 +237,22 @@
 "","","","","","","","","","","","base","refund","-82||-86||+84","","","",""
 "","","","","","","","","","","","tax","refund","","a411","","",""
 "","","","","","","","","","","","tax","refund","","a451055","-100","",""
+"attn_VAT-IN-V82-21-EU-G-D50","1223","","21%","21% EU G D50","False","21.0","percent","purchase","tax_group_tva_21","False","base","invoice","+82||+86","","","",""

```

Comment:
not sure but in the excalidraw the name is like this
```suggestion
"attn_VAT-IN-V82-21-EU-G-ND50","1223","","21%","21% EU G ND50","False","21.0","percent","purchase","tax_group_tva_21","False","base","invoice","+82||+86","","","",""

```
(and updated the xmlid accordingly)